dutch/bos/ontvoering/politie/politie.md

```diff
@@ -3,4 +3,8 @@ Eenmaal in verbinding met de hulpdiensen, vertel je je locatie.
 Je legt het probleem uit en de politie is meteen onderweg.
 Eind goed al goed!
 
-Of toch niet...
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.